libs/auth/src/certs.ts

```diff
@@ -44,23 +47,28 @@ export const STANDARD_CERT_FIELDS = [
 
 interface VxAdminCustomCertFields {
   component: 'admin';
+  machineId: string;
   jurisdiction: string;
 }
 
 interface VxCentralScanCustomCertFields {
   component: 'central-scan';
+  machineId: string;
 }
 
 interface VxMarkCustomCertFields {
   component: 'mark';
+  machineId: string;
 }
 
 interface VxMarkScanCustomCertFields {
   component: 'mark-scan';
+  machineId: string;
 }
 
 interface VxScanCustomCertFields {
   component: 'scan';
+  machineId: string;
```

Comment:
should these interfaces now extend a base interface?

Reply:
https://github.com/votingworks/vxsuite/pull/5557/commits/4ee780457b37dafeac6a0509ffde989f9075f6dc